File: P007/main.java
import java.lang.*;

public class main {
    private static boolean isPrime(int n) {
        for(int i=2; i<=(int)Math.sqrt(n/2); i++)
            if(n%i == 0)
                return false;
        return true;
    }

    private static void findPrimeNumber(int n) {
        int result = 0;
        int ctr = 0;
        int start = 2;

        while(ctr != n) {
            if(isPrime(start)) {
                result = start;
                ctr++;
            }
            start++;
        }
        System.out.println(result);
    }


    public static void main(String[] args) {
        long startTime = System.currentTimeMillis();

        findPrimeNumber(10001);

        long endTime = System.currentTimeMillis();
        long totalSpan = System.currentTimeMillis();
        System.out.println("Completed in " + totalSpan + " milliseconds.");
    }
}

File: P007/main.py
import time

start = time.time()

def isPrime(n) :
    for i in range(2, n/2):
        if n%i == 0 :
            return False

    return True

def findPrimeNumber(n) :
    result = 0
    ctr = 0
    start = 2

    while ctr != n:
        if isPrime(start) :
            result = start
            ctr += 1
        start += 1

    return result

print findPrimeNumber(10001)

elapsed = time.time()

print("found in %s seconds") % (elapsed)
